core/application/impl/validating_node_application.cpp

```diff
@@ -15,10 +15,11 @@ namespace kagome::application {
         logger_(common::createLogger("Application")) {
     spdlog::set_level(app_config->verbosity());
 
-    // genesis launch if database does not exist
-    babe_execution_strategy_ = boost::filesystem::exists(app_config->leveldb_path())
-                      ? Babe::ExecutionStrategy::SYNC_FIRST
-                      : Babe::ExecutionStrategy::GENESIS;
+    if (not app_config->is_already_synchronized()) {
+      babe_execution_strategy_ = Babe::ExecutionStrategy::SYNC_FIRST;
+    } else {
+      babe_execution_strategy_ = Babe::ExecutionStrategy::START;
+    }
 
```

Comment:
```suggestion
    if (app_config->is_already_synchronized()) {
      babe_execution_strategy_ = Babe::ExecutionStrategy::START;
    } else {
          babe_execution_strategy_ = Babe::ExecutionStrategy::SYNC_FIRST;
    }
```
Negations are a bit harder to read, so I hope you don't mind this tiny change.

Reply:
agree